src/app/global/components/nav-desktop/nav-desktop.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+interface Item {
+  icon: string;
+  rota: string;
+  href: string;
+}
+
+@Component({
+  selector: 'app-nav-desktop',
+  templateUrl: './nav-desktop.component.html',
+  styleUrls: ['./nav-desktop.component.scss'],
+})
+export class NavDesktopComponent implements OnInit {
```

Comment:
esse componente não precisa implementar o OnInit
```suggestion
export class NavDesktopComponent {
  public items: Item[] = [
    { icon: 'forms_add_on', rota: 'Formulário', href: '/formulario' },
    { icon: 'group', rota: 'Pacientes', href: '/pacientes' },
    { icon: 'folder_shared', rota: 'Tratamentos', href: '/tratamentos' },
    { icon: 'badge', rota: 'Trabalhadores', href: '/trabalhadores' },
    { icon: 'settings', rota: 'Configurações', href: '/configuracoes' },
  ];

  public navFixed: boolean = true;

  @Output() newItemEvent = new EventEmitter<boolean>();

  toggleNav(): void {
    this.navFixed = !this.navFixed;
    this.newItemEvent.emit(this.navFixed);
  }
}
```